packages/contract/README.md

```diff
@@ -10,51 +10,50 @@ $ npm install @truffle/contract
 
 ### Features
 
-* Synchronized transactions for better control flow (i.e., transactions won't finish until you're guaranteed they've been mined).
-* Promises. No more callback hell. Works well with `ES6` and `async/await`.
-* Default values for transactions, like `from` address or `gas`.
-* Returning logs, transaction receipt and transaction hash of every synchronized transaction.
+- Synchronized transactions for better control flow (i.e., transactions won't finish until you're guaranteed they've been mined).
+- Promises. No more callback hell. Works well with `ES6` and `async/await`.
+- Default values for transactions, like `from` address or `gas`.
+- Returning logs, transaction receipt and transaction hash of every synchronized transaction.
 
 ### Usage
 
 First, set up a new web3 provider instance and initialize your contract, then `require("@truffle/contract")`. The input to the `contract` function is a JSON blob defined by [@truffle/contract-schema](https://github.com/trufflesuite/truffle/tree/master/packages/contract-schema). This JSON blob is structured in a way that can be passed to all truffle-related projects.
 
 ```javascript
-var provider = new Web3.providers.HttpProvider("http://localhost:8545");
-var contract = require("@truffle/contract");
+const provider = new Web3.providers.HttpProvider("http://localhost:8545");
+const contract = require("@truffle/contract");
 
-var MyContract = contract({
-  abi: ...,
-  address: ..., // optional
-  // many more
-})
+// Artifact json previosly saved by @truffle/artifactor
+const contractArtifact = require("./path/to/contractArtifact.json");
+
+const MyContract = contract(contractArtifact);
```

Comment:
This json blob threw me off and made me think I could simply pass the ABI without running it through the artifactor.

Reply:
it would be nice if we could explicitly define what the minimum is that you'll need for an artfact - and the consequences of including less information than the entire artifactor output

Reply:
Second Tyler's suggestion

Reply:
If we're referring to the shape of the JSON `contract` needs I think it's worth noting in the documentation. I suspect it will at least match [what web3's contract abstraction needs](https://web3js.readthedocs.io/en/v1.7.0/glossary.html#glossary-json-interface)

Reply:
My understanding is the absolute *minimum* you need is an ABI.  The problem with showing that, though, is that it might confuse people into passing `abi` instead of `{ abi }`, the exact problem we're trying to solve!

Reply:
This PR came about because I attempted to pass: 
```
{
  abi: ...,
  address: ..., 
}
```
as the old README suggested and it didn't work.  It was about a month ago at this point, and it's very possible I was doing something wrong, though.  I'll try it again and see if I hit the same issue.

Reply:
I tried passing only the abi and it worked, so it was definitely user error on my part.  Instead of removing the original example, I added a second on here to show that you can additionally use the output of the artifactor.  Let me know if any of you think of a more concise way of explaining this.